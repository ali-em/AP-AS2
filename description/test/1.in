98821
10011114449223343300088888888
