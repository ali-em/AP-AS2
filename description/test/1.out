9821
10149234308
